prime-router/src/main/kotlin/LivdTableDownload.kt

```diff
@@ -0,0 +1,98 @@
+package gov.cdc.prime.router
+
+import it.skrape.core.htmlDocument
+import it.skrape.fetcher.HttpFetcher
+import it.skrape.fetcher.response
+import it.skrape.fetcher.skrape
+import it.skrape.selects.eachHref
+import it.skrape.selects.html5.a
+import java.io.File
+import java.io.FileOutputStream
+import java.net.URL
+import java.util.Scanner
+
+private const val cdcLOINCTestCodeMappingPage = "https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html"
+private const val livdSARSCov2File = "LIVD-SARS-CoV-2"
+
+/**
+ *  The LivdTable class downloads the latest LOINC test data, so it can be ingested automatically.
+ *  It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html.
+ *  If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
+ *  If the option is not specified, it will download the file to ./build directory.
+ */
+data class LivdTable(val outputDir: String) {
```

Comment:
Remove data from data class.  Not a data class.  Also, if you keep this class, consider a name that is a bit more descriptive of its function.  E.g. LivdTableDownloader?

Reply:
Done.  Thanks